src/index.js

```diff
@@ -44,7 +44,11 @@ class TestnetNode {
     return Promise.all(promises)
   }
   stop () {
-    return Promise.resolve()
+    const promises = []
+    if (this.config.btp) {
+      promises.push(this.pluginFactory.stop())
```

Comment:
If there's only one promise, why use an array and `Promise.all()`?

Reply:
I agree it's currently superfluous, but I did it this way for symmetry with the start() function. Also in case we want to add a this.removePlugin function in the future.